tokenserver-settings/src/lib.rs

```diff
@@ -74,12 +75,12 @@ pub struct Jwk {
 impl Default for Settings {
     fn default() -> Settings {
         Settings {
-            database_url: "mysql://root@127.0.0.1/tokenserver_rs".to_owned(),
+            database_url: "mysql://root@127.0.0.1/tokenserver".to_owned(),
             database_pool_max_size: None,
             database_pool_min_idle: None,
             database_pool_connection_timeout: Some(30),
             enabled: false,
-            fxa_email_domain: "api.accounts.firefox.com".to_owned(),
+            fxa_email_domain: "api-accounts.stage.mozaws.net".to_owned(),
```

Comment:
Is this temporary, or do we always want to point to the stage server as the default?

Reply:
I think it would make sense always to point to stage as the default. That way, in dev environments, people don't mistakenly hit prod if the necessary settings are unset. (The rest of the settings in this file already pointed to stage -- I just updated this one to make it consistent with the others.) Open to other suggestions though!